app/controllers/hub/clients_controller.rb

```diff
@@ -47,8 +43,6 @@ def destroy
     end
 
     def edit
-      return render "public_pages/page_not_found", status: 404 if @client.intake.is_ctc?
```

Comment:
this "public_pages/page_not_found" does not actually exist in the hub so this page is right now showing up blank, "raise CanCan::AccessDenied" will show the Access Denied page